### ISSUE TITLE
404 when saving submission from non-primary site

### ISSUE THREAD

**Issue by jmauzyk** [verbb/formie#951]
**Description**
When saving a form submission that was received on the primary Craft site, the URL follows the format 
`https://mysite.com/admin/formie/submissions/form/123456?site=primary`
before saving, and after saving redirects to
`https://mysite.com/admin/formie/submissions/form/123456`

However, when saving a submission that was received from a non-primary Craft site, the URL redirects from
`https://mysite.com/admin/formie/submissions/form/234567?site=non_primary`
to 
`https://mysite.com/admin/formie/submissions/form/234567/non_primary`
which results in a 404

**Steps to reproduce**
1. Go to CP edit page for a submission created through a site not marked as primary.
2. Click Save.
3. Returns 404.

### Form settings
Please pick from the options below.

- Multi-page form: No
- Submission Method: Page Reload
- Client-side Validation: Yes
- Custom Form Templates: No

**Additional info**
- Plugin version: 1.6.1
- Craft version: Pro 3.7.45
- Multi-site: Yes

**Comment by engram-design** [comment]
Should be fixed for the next release. To get this fix early, change your `verbb/formie` requirement in `composer.json` to:

```
"require": {
  "verbb/formie": "dev-craft-3 as 1.6.1",
  "...": "..."
}
```

Then run `composer update`.

**Comment by engram-design** [comment]
Fixed in [1.6.2](https://github.com/verbb/formie/releases/tag/1.6.2)
